README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
You have test the method's setter behaviour, but not the change for fixer behaviour.
Check this out: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/30acb92898d474e5caa4da5314d281e889889f9f/Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php#L36